server/src/main/java/org/elasticsearch/gateway/LocalAllocateDangledIndices.java

```diff
@@ -277,19 +276,11 @@ public void writeTo(StreamOutput out) throws IOException {
 
     public static class AllocateDangledResponse extends TransportResponse {
 
-        private AllocateDangledResponse(StreamInput in) throws IOException {
-            if (in.getTransportVersion().before(TransportVersions.V_8_0_0)) {
-                in.readBoolean();
-            }
-        }
+        private AllocateDangledResponse(StreamInput in) throws IOException {}
```

Comment:
So this payload is just... empty? Presumably this response type is probably either unused or not relevant anymore at all. Should we make a not to come back and remove this?